Documentation/QuickInstall/Composer/Index.rst

```diff
@@ -6,17 +6,17 @@
 Install TYPO3 via composer
 ================================
 
-The recommended way of installing TYPO3 is via composer.
+The recommended way of installing TYPO3 is via Composer.
 
 To create a new TYPO3 project use the TYPO3 Base Distribution:
 
 .. code-block:: bash
 
 	# Download the Base Distribution, the latest "stable" release (8.7)
-	composer create-project typo3/cms-base-distribution YourNewProjectFolder
+	composer create-project typo3/cms-base-distribution:^8 YourNewProjectFolder
```

Comment:
There is a mismatch between "latest stable" and 8.7. To get the latest stable, don't specify the version number. To get a specific version, specify it

Reply:
Resolved this with a wording fix now.